src/processors/models/database/cassandra/cassandraConnector.js

```diff
@@ -5,17 +5,32 @@ let client;
 
 const INSERT_PROCESSOR = 'INSERT INTO processors(id, name, description, javascript, created_at, updated_at) values(?,?,?,?,?,?)';
 const GET_ALL_PROCESSORS = 'SELECT * FROM processors';
-const GET_PROCESSOR = 'SELECT * FROM processors WHERE id=?';
+const GET_PROCESSOR_BY_ID = 'SELECT * FROM processors WHERE id=?';
 const DELETE_PROCESSOR = 'DELETE FROM processors WHERE id=?';
 const UPDATE_PROCESSOR = 'UPDATE processors SET name=?, description=?, javascript=?, updated_at=? WHERE id=? AND created_at=? IF EXISTS';
 
+const INSERT_PROCESSOR_MAPPING = 'INSERT INTO processors_mapping(name, id) VALUES(?, ?)';
+const DELETE_PROCESSOR_MAPPING = 'DELETE FROM processors_mapping WHERE name=?';
+const GET_PROCESSOR_MAPPING = 'SELECT * FROM processors_mapping WHERE name=?';
```

Comment:
missing `UPDATE_PROCESSOR_MAPPING` that should be used under function `updateProcessor` if the name is updated

Reply:
Cassandra doesn't support updating a record's primary key, therefore I need to delete and insert a new key-value record